src/custom/state/orders/helpers.tsx

```diff
@@ -46,7 +50,35 @@ type MetaPopupContent = GPPopupContent<OrderTxTypes.METATXN>
 type TxnPopupContent = GPPopupContent<OrderTxTypes.TXN>
 
 function setOrderSummary({ id, summary, status, descriptor }: SetOrderSummaryParams) {
-  return summary ? `${summary} ${descriptor || status}` : `Order ${formatOrderId(id)} ${descriptor || status}`
+  return !summary
+    ? `Order ${formatOrderId(id)} ${descriptor || status || ''}`
+    : typeof summary === 'string'
+    ? `${summary} ${descriptor || status || ''}`
+    : summary
+}
+
+const Wrapper = styled.div`
+  & > p:first-child {
+    margin-top: 0;
+  }
+
+  & > p:last-chid {
+    margin-bottom: 0;
+  }
+`
+
+export function buildCancellationPopupSummary(id: string, summary: string | undefined): JSX.Element {
+  // TODO: style this!
+  // TODO: this probably shouldn't be here anyway :shrug:
```

Comment:
I think styles are mostly fine. So maybe we should remove this? or u want to make a suggestion in an issue? 

One thing that we might want to improve, is the cancel button, although I guess is also kind of fine. But the popup looks great to me.